frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx

```diff
@@ -62,15 +62,13 @@ function CellInfoSideBar({
     track(EVENTS.WMG_FMG_ADD_GENES_CLICKED);
   }, [data, dispatch]);
 
-  const [hoverTime, setHoverTime] = useState({ start: -1, end: -1 });
+  const [hoverStartTime, setHoverStartTime] = useState(0);
 
-  useEffect(() => {
-    const { start, end } = hoverTime;
-    if (start !== -1 && end !== -1 && end - start > 2 * 1000) {
+  const handleHoverEnd = useCallback(() => {
+    if (Date.now() - hoverStartTime > 2 * 1000) {
```

Comment:
super nit: cognitively adding is easier to comprehend than subtracting, should we do `Date.now() > hoverStartTime + 2 * 1000` ? Personally it reads better (current time is larger than hoverStartTime plus 2 seconds vs. current time minus hoverStartTime is larger than 2 seconds)

Also should we extract magic number 2 * 1000 as a constant?

Reply:
I believe the standard for calculating time duration between saved point and now is: ```Date.now() - savedTime```, I can pull this out into a named variable if that helps

Reply:
```
Date.now() > hoverStartTime + 2 * 1000
```

Requires much more cognitive load for me than
```
Date.now() - savedDate > 2 * 1000
```

because I instantly pattern match 
```
Date.now() - savedDate
``` 
to `duration`

Reply:
Ahh got it, I see what you're seeing now. Thanks for explaining. No need to extract a variable `duration` thank you!